cmd/dance/main.go

```diff
@@ -119,7 +120,21 @@ func main() {
 			log.Fatal(err)
 		}
 
-		runRes, err := gotest.Run(ctx, dir, config.Args, *vF)
+		var runRes *internal.TestResults
+		switch config.Runner {
+		case "gotest":
+			runRes, err = gotest.Run(ctx, dir, config.Args, *vF)
+		case "jstest":
+			err = jstest.Run(ctx, config.Args)
```

Comment:
I think we should make runner return results. We can treat each file/argument as a test

Reply:
I used the `internal.TestResult` type for now just to see if it would work. Should we treat this runner and its output separately and not use the same types? Given that the output is different etc.

Reply:
I think we need to also parse stdout for assertions etc. to have detailed results.

Reply:
No, runners should share result types and config type, that's the point.

I think there is no need to parse stdout/stderr yet, just return it